src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -276,51 +262,4 @@ export default class Sidewinder extends Module {
 		}
 		return rawDamage
 	}
-
-	_addInSingleTargetShadowbites() {
```

Comment:
I'm gonna trust you on this lot :meowsweats:

Reply:
This function was able to be obsoleted by determining if a shadowbite was ST based on the normaliseddamage hits property, so this module doesn’t have to do the event deduplication on its own any more.